python/ql/test/library-tests/taint/strings/test.py

```diff
@@ -14,6 +14,7 @@ def test_list(x, y, z):
     b = tainted_list[x]
     c = tainted_list[y:z]
     d = tainted_list.copy()
+    e, f, g = tainted_list # TODO: currently not handled
```

Comment:
Did you want to extend this example so that the variables get used subsequently?

Reply:
from my new experience, then YES! :smile: